include/avif/avif.h

```diff
@@ -551,7 +551,17 @@ typedef enum avifRGBFormat
     AVIF_RGB_FORMAT_ARGB,
     AVIF_RGB_FORMAT_BGR,
     AVIF_RGB_FORMAT_BGRA,
-    AVIF_RGB_FORMAT_ABGR
+    AVIF_RGB_FORMAT_ABGR,
+    // RGB_565 format uses five bits for the red and blue components and six
+    // bits for the green component. Each RGB pixel is 16 bits (2 bytes) are
```

Comment:
Nit: should "are" be changed to "and"?

Or perhaps this sentence should read:

> Each RGB pixel is 16 bits (2 bytes), which are packed as follows:

Reply:
Done.